runtime/vm/exceptionsupport.c

```diff
@@ -940,6 +940,77 @@ setClassLoadingConstraintSignatureError(J9VMThread *currentThread, J9ClassLoader
 }
 
 
+void  
+setClassLoadingConstraintOverrideError(J9VMThread *currentThread, J9UTF8 *newClassNameUTF, J9ClassLoader *loader1, J9UTF8 *class1NameUTF, J9ClassLoader *loader2, J9UTF8 *class2NameUTF, J9UTF8 *exceptionClassNameUTF, U_8 *methodName, UDATA methodNameLength, U_8 *signature, UDATA signatureLength)
+{
+	PORT_ACCESS_FROM_VMC(currentThread);
+	char * msg = NULL;
+	const char * nlsMessage;
```

Comment:
nitpick: initial value?  Can we directly assign the `j9nls_` call to this?

Reply:
Yes, this was copied code.

Reply:
Fixed all similar occurrences.